ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -157,11 +157,20 @@ do
   yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "$SHARED_DIR/install-config.yaml" "$f"
 done
 
+if [ "${FIPS_ENABLED:-false}" = "true" ]; then
+	echo "Adding 'fips: true' to install-config.yaml"
+	yq --yaml-output --in-place ".
+		| .fips = true
+	" "$SHARED_DIR/install-config.yaml"
+fi
+
 mkdir -p "${INSTALL_DIR}"
 cp "${SHARED_DIR}/install-config.yaml" "${INSTALL_DIR}/"
 # From now on, we assume no more patches to the install-config.yaml are needed.
 # We can create the installation dir with the manifests and, finally, the ignition configs
 
+export OPENSHIFT_INSTALL_SKIP_HOSTCRYPT_VALIDATION=true
```

Comment:
It would be applicable to all installations, so it would be better to include it with the condition fips: true as below.
```suggestion
if [ "${FIPS_ENABLED:-false}" = "true" ]; then
    export OPENSHIFT_INSTALL_SKIP_HOSTCRYPT_VALIDATION=true
fi
```

Reply:
@mhanss Please see this note from above:

Something weird is happening. exporting OPENSHIFT_INSTALL_SKIP_HOSTCRYPT_VALIDATION by running:
if [ "${FIPS_ENABLED:-false}" = "true" ]; then
# Don't require the installer to run in a FIPS-enabled environment
export OPENSHIFT_INSTALL_SKIP_HOSTCRYPT_VALIDATION=true
fi

is not working

it's only taking effect when outside if statement:
export OPENSHIFT_INSTALL_SKIP_HOSTCRYPT_VALIDATION=true

Reply:
It should be present in the directory from which you are executing the `openshift-install` command to create a cluster, so it should be added to baremetal-lab-ipi-install-commands.sh.

Reply:
It seems you have not added the FIPS_ENABLED variable to the ref file.

Reply:
@mhanss Deployment passed with above changes:
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/52984/rehearse-52984-periodic-ci-openshift-openshift-tests-private-release-4.16-multi-nightly-baremetal-ipi-ovn-ipv4-fips-amd-vmedia-f7/1803254232447455232